### DISCUSSION TITLE
windows版本的服务假死，不接收消息，需要回到cmd界面按回车一次才正常有人知道怎么解决吗

### DISCUSSION THREAD

**Discussion by 1447262390** [Mrs4s/go-cqhttp#1801] [Q&A]
版本：v1.0.0-rc3
操作系统：win10
问题：运行的时候突然假死，无法获取到QQ消息了，不定时，有时能跑一个晚上，有时几分钟就会，每次假死需要在cmd界面按回车才能继续接收消息，假死的时候，我看了一下端口的连接状态，发现5701都是wait_time状态，按了回车之后再查看端口状态，就发现有回复正常的了，这个问题出现非常频繁，苦恼没有解决办法，请教

**Comment by 1447262390** [comment]
windows环境运行不稳定，直接换到linux用docker打包跑了

**Comment by XilyFeAAAA** [comment]
我也遇到了这个问题，请问windows环境有解决办法吗

**Reply by 1447262390** [reply]
暂时没有解决办法，我在linux上跑之后，就再也没出现过这种问题了
